Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -244,6 +244,7 @@ namespace gdjs {
     contactsStartedThisFrame: Array<Physics2RuntimeBehavior>;
```

Comment:
can you comment for each of these arrays its responsibility (what does it hold, when it's updated)?